docs/optimization_guide/dldt_optimization_guide.md

```diff
@@ -11,30 +11,18 @@ Performance means how fast the model is in deployment. Two key metrics are used
 
 Latency measures inference time (ms) required to process a single input. When it comes to batch input need to measure throughput (images per second or frames per second, FPS). To calculate throughput, divide number of frames that were processed by the processing time.   
 
-> **NOTE**: To get performance numbers for OpenVINO, as well as tips how to measure it and compare with native framework, check [Getting performance numbers](../MO_DG/prepare_model/Getting_performance_numbers.md) page.
+## How to measure performance
+To get performance numbers for OpenVINO, as well as tips how to measure it and compare with native framework, check [Getting performance numbers](../MO_DG/prepare_model/Getting_performance_numbers.md) page.
  
 ## How to Improve Performance 
 
-> **NOTE**: Make sure that your model can be successfully inferred with OpenVINO Inference Engine. 
+> **NOTE**: Make sure that your model can be successfully inferred with OpenVINO Inference Engine before diving into optimization topic. 
```

Comment:
Make sure that your model can be successfully inferred with OpenVINO Inference Engine before referring to the optimization topic.